### ISSUE TITLE
Support Forge 1.20.1

### ISSUE THREAD

**Issue by QarthO** [abc123me/BetterForgeChat#14]
Any chance you could update this to support minecraft forge 1.20.1? running a ATM9 modpack and would love these features

**Comment by abc123me** [comment]
No promises, but since I get asked literally all the time there's a "main-1.20.x" branch I just created for it, after the weekend, if you remind me by replying, I'll work on it.

Also, it will **NOT** support Mojang's chat reporting crap.

**Comment by verybutter** [comment]
i'd like to put interest forward on this as well :)

**Comment by loander** [comment]
would love to see 1.20.1 release as well

**Comment by QarthO** [comment]
> No promises, but since I get asked literally all the time there's a "main-1.20.x" branch I just created for it, after the weekend, if you remind me by replying, I'll work on it.
> 
> Also, it will **NOT** support Mojang's chat reporting crap.

Reminding as requested!

**Comment by abc123me** [comment]
In started working on it, I've got some improvements and a new version (V1.3.0) in my local repo I need to get integrated first, after that I should be able to start on 1.20 support.

**Comment by SuperDevyn** [comment]
THANK YOU. I just recently started a modpack server and I desperately need this mod for 1.20.1

If there's anyway I could support you let me know and I'd be happy to.

**Comment by creaper3** [comment]
tysm it would be amazing if you could update this. There is currently no mod for 1.20.1 that does the things this mod does, not even with fabric connectors.

**Comment by Baomont** [comment]
Is this happening!? I can promise people will be very hyped!

**Comment by abc123me** [comment]
Finding the motivation is really hard, especially when I actively hate playing newer versions of MC since as the game has gotten much worse (in my opinion). Not to mention as a freelancer managing this mod is a huge waste of time as I've probably invested well over 50 hours into it and made nothing but more problems for myself. MY going rate for freelance work (embedded software engineering) is usually around $50/hour, that means this mod has costed me $2500 to develop, and I no longer have an active server (that I actually enjoy playing) to develop it on. Originally this mod was made for 1.18.2 since me and some friends wanted to play a custom modpack we made and opened up the modpack and server to everyone in our college, that kinda died off a while ago and now frankly I just don't see a point in booting up and working on the mod when i've got tens of hundreds of far more engaging and rewarding projects I could be doing.

TLDR; I don't see a point in writing a shitty mod for a shitty game that I don't enjoy anymore when I have hundreds of other more fulfilling projects I could be doing.

**Comment by abc123me** [comment]
Update, somebody helped me out some and this has entered development some:
https://github.com/abc123me/BetterForgeChat/pull/18

I'm gonna start putting together a release soon.

**Comment by yaroslav4167** [comment]
I build version for commit https://github.com/abc123me/BetterForgeChat/commit/dbf6fcefb12561cb74b2fa015cecaccb16f58def and it is worked! 
If anyone needs it, I will attach the builded version: [bfcmod-1.20.1-2.0.zip](https://github.com/user-attachments/files/17090805/bfcmod-1.20.1-2.0.zip)
